backend/data-store/db-client/submit.go

```diff
@@ -0,0 +1,61 @@
+package dbclient
```

Comment:
I think the file can be renamed to some more generic name. Maybe `update`?

Reply:
I'd rather not call this update - it's very similar to `CRUD` typical but queries in `read.go` refer to some queries from `create.go` (hence they are not READ_ONLY); 
At the same time we have `UpsertPartialRatings` which is sort of in between. Seeing as the submit query does some business logic i think it warrants it's own file with a bit more specific name

Reply:
Okay, I see now. Let's keep it then.